docs/logs/complex-objects/FoodRecallNotice.cs

```diff
@@ -1,15 +1,15 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-public class FoodRecallNotice
+public struct FoodRecallNotice
```

Comment:
would you show the benchmarks results from this change?

Reply:
|           Method |     Mean |   Error |   StdDev |  Gen 0 | Allocated |
|----------------- |---------:|--------:|---------:|-------:|----------:|
| LogPayloadClass  | 122.1 ns | 3.45 ns | 9.67 ns  | 0.0088 |      56 B |
| LogPayloadStruct | 122.6 ns | 3.60 ns | 10.44 ns |      - |         - |

Reply:
I think the only noticeable difference is heap vs. stack alloc (CPU is almost the same).